src/rp2_common/hardware_rosc/rosc.c

```diff
@@ -58,4 +58,12 @@ void rosc_set_dormant(void) {
     rosc_write(&rosc_hw->dormant, ROSC_DORMANT_VALUE_DORMANT);
     // Wait for it to become stable once woken up
     while(!(rosc_hw->status & ROSC_STATUS_STABLE_BITS));
-}
\ No newline at end of file
+}
+
+void rosc_restart(void) {
```

Comment:
Is it specifically a "restart", or could this function be named `rosc_enable` in order to mirror `rosc_disable` ?